data/json/species.json

```diff
@@ -59,11 +59,18 @@
     "description": "a nether creature",
     "fear_triggers": [ "HURT" ]
   },
+  {
+    "type": "SPECIES",
+    "id": "NETHER_EMANATION",
+    "description": "a reflection of another dimension",
+    "fear_triggers": [ "HURT" ]
+  },
   {
     "type": "SPECIES",
     "id": "MIGO",
     "description": "a creature from the mi-go homeworld",
-    "fear_triggers": [ "HURT" ]
+    "fear_triggers": [ "HURT" ],
+    "bleeds": "fd_blood_invertebrate"
```

Comment:
did you mean to sneak this in?

Reply:
Oops, nope that was something I was thinking about doing and thought I'd backed out once I realized it was out of scope.